### PR TITLE
Update deploy.md

### PR DESCRIPTION
**Pull request by MaxiReglisse**
it seems to me that the geowebcache-webapp was forgotten... that's right ??

Ernest.

### CODE REVIEW

**Comment by fvanderbiest** [comment]
You're totally right !

File: doc/deploy.md [hunk 1]
```diff
@@ -27,6 +27,7 @@ Once your system is ready, collect WAR files in a dedicated directory and rename
     mv ldapadmin-${VERSION}-${PROFILE}.war ldapadmin.war
     mv mapfishapp-${VERSION}-${PROFILE}.war mapfishapp.war
     mv header-${VERSION}-${PROFILE}.war header.war
+    mv geowebcache-webapp-${VERSION}-${PROFILE}.war $WEBAPPS/geowebcache.war
```
Comment (fvanderbiest) [inline doc/deploy.md]: $WEBAPPS ?

File: doc/deploy.md [hunk 2]
```diff
@@ -76,6 +77,7 @@ However, this requires the build process to happen on the production host (or a
     sudo ln -s ~/.m2/repository/org/georchestra/ldapadmin/${VERSION}-SNAPSHOT/ldapadmin-${VERSION}-SNAPSHOT-${PROFILE}.war ldapadmin.war
     sudo ln -s ~/.m2/repository/org/georchestra/mapfishapp/${VERSION}-SNAPSHOT/mapfishapp-${VERSION}-SNAPSHOT-${PROFILE}.war mapfishapp.war
     sudo ln -s ~/.m2/repository/org/georchestra/header/${VERSION}-SNAPSHOT/header-${VERSION}-SNAPSHOT-${PROFILE}.war header.war
+    sudo ln -s ~/.m2/repository/org/georchestra/geowebcache-webapp/${VERSION}-SNAPSHOT/header-${VERSION}-SNAPSHOT-${PROFILE}.war geowebcache.war
```
Comment (fvanderbiest) [inline doc/deploy.md]: header-${VERSION}-SNAPSHOT-${PROFILE}.war ?

File: doc/deploy.md (hunk 1 above)
Comment (MaxiReglisse) [inline doc/deploy.md]: oups !

WEBAPPS=/var/lib/tomcat6/webapps in my configuration. but it is not useful... i will correct that.

File: doc/deploy.md (hunk 2 above)
Comment (MaxiReglisse) [inline doc/deploy.md]: shit... ! :(

**Comment by fvanderbiest** [comment]
Looks good. Merging.

But I will add a comment saying that a standalone GeoWebCache is **very** optional, since there is already one which ships with GeoServer (aka "the integrated one")

**Comment by MaxiReglisse** [comment]
sorry, but i did not know that...

**Comment by fvanderbiest** [comment]
np.

Comment added with https://github.com/georchestra/georchestra/commit/9697837dfd7f8eda017649d89bb1a13f137740d1

**Pull request merged**